cwl/train.cwl

```diff
@@ -52,7 +52,7 @@ inputs:
       prefix: --model
   first_round:
     type: boolean
-    doc: Wether it is the first aggregation round or not
+    doc: Whether it is the first aggregation round or not
```

Comment:
One of the English words I know exists, but I have no equivalent in Portuguese for that :rofl: 

https://www.dictionary.com/browse/wether

Reply:
Hahaha. In Italian we just call it "Castrato"

Reply:
In Portuguese too, castrado, but in English there's castrated for that (and neutered and spayed. This wether is specific for male sheep (as if the suffering wasn't enough, they've given it a special name).